src/main/sensors/gyro.c

```diff
@@ -746,15 +747,34 @@ static bool isDynamicFilterActive(void)
     return feature(FEATURE_DYNAMIC_FILTER);
 }
 
-static void gyroInitFilterDynamicNotch(gyroSensor_t *gyroSensor)
+static void gyroInitFilterDynamic(gyroSensor_t *gyroSensor)
 {
-    gyroSensor->notchFilterDynApplyFn = nullFilterApply;
+    gyroSensor->dynFilterApplyFn = nullFilterApply;
 
     if (isDynamicFilterActive()) {
-        gyroSensor->notchFilterDynApplyFn = (filterApplyFnPtr)biquadFilterApplyDF1; // must be this function, not DF2
-        const float notchQ = filterGetNotchQ(400, 390); //just any init value
-        for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-            biquadFilterInit(&gyroSensor->notchFilterDyn[axis], 400, gyro.targetLooptime, notchQ, FILTER_NOTCH);
+        switch (gyroConfig()->dyn_filter_type) {
+        case FILTER_PT1: {
+            const int dynamicLowpassDefaultCutoffHz = 200;
+            const float gyroDt = gyro.targetLooptime * 1e-6f;
+            const float gain = pt1FilterGain(dynamicLowpassDefaultCutoffHz, gyroDt);
+
+            gyroSensor->dynFilterApplyFn = (filterApplyFnPtr) pt1FilterApply;
+
+            for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
+                pt1FilterInit(&gyroSensor->dynFilter[axis].pt1FilterState, gain);
+            }
+        } break;
```

Comment:
The formatting with closing bracket and `break` on the same line seems to be a bit weird.

Reply:
Will change then